cloudinit/config/cc_apk_configure.py

```diff
@@ -102,104 +102,7 @@
-                                mirror, to download official packages from.
-                                If not specified then it defaults to ``{}``
-                            """.format(
-                                    DEFAULT_MIRROR
-                                )
-                            ),
-                        },
-                        "community_enabled": {
-                            "type": "boolean",
-                            "default": False,
-                            "description": dedent(
-                                """\
-                                Whether to add the Community repo to the
-                                repositories file. By default the Community
-                                repo is not included.
-                            """
-                            ),
-                        },
-                        "testing_enabled": {
-                            "type": "boolean",
-                            "default": False,
-                            "description": dedent(
-                                """\
-                                Whether to add the Testing repo to the
-                                repositories file. By default the Testing
-                                repo is not included. It is only recommended
-                                to use the Testing repo on a machine running
-                                the ``Edge`` version of Alpine as packages
-                                installed from Testing may have dependancies
-                                that conflict with those in non-Edge Main or
-                                Community repos."
-                            """
-                            ),
-                        },
-                        "version": {
-                            "type": "string",
-                            "description": dedent(
-                                """\
-                                The Alpine version to use (e.g. ``v3.12`` or
-                                ``edge``)
-                            """
-                            ),
-                        },
-                    },
-                    "required": ["version"],
-                    "minProperties": 1,
-                    "additionalProperties": False,
-                },
-                "local_repo_base_url": {
-                    "type": "string",
-                    "description": dedent(
-                        """\
-                        The base URL of an Alpine repository containing
-                        unofficial packages
-                    """
-                    ),
-                },
-            },
-            "minProperties": 1,  # Either preserve_repositories or alpine_repo
-            "additionalProperties": False,
-        }
-    },
-}
-
-__doc__ = get_meta_doc(meta, schema)
+__doc__ = get_meta_doc(meta)
```

Comment:
[_get_meta_doc now makes schema param optional and will construct docs based on sub-schema matching meta['id']](https://github.com/canonical/cloud-init/pull/1175/files#diff-3dab74160d0d2e18837e4c271eb03209be4263cf51d44cb8ca78bb739fd3f68cR584) for the "new-style" cc_ modules.